guides/common/modules/proc_provisioning-from-a-builder-image-on-smartproxy.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="provisioning-from-a-builder-image-on-{smart-proxy-context}"]
+= Provisioning from a {LoraxCompose} image on {SmartProxy}
+
+You can upload a {LoraxCompose} image to {Project} and use this image to provision hosts by using network boot.
+
+ifndef::satellite[]
+[NOTE]
+====
+This feature is available for {EL} hosts only.
+====
+endif::[]
+
+When you provision from a {LoraxCompose} image, Anaconda installer partitions disks, downloads and mounts the image, and copies files over to a host.
+
+.Prerequisites
+* An existing TAR image created by using {LoraxCompose} in {the-Cockpit}.
+Ensure that your blueprint to build the image includes the xref:packages-required-in-a-builder-image[required packages].
+* If you want to use {SmartProxyServer} URL, then the referenced lifecycle environment, such as Library, must be synchronized to that {SmartProxyServer}.
+For more information, see {InstallingSmartProxyDocURL}Adding_Lifecycle_Environments_{smart-proxy-context}[Adding lifecycle environments to {SmartProxyServers}] in _{InstallingSmartProxyDocTitle}_.
+
+.Procedure
+. On {Project}, create a {customproduct}, add a {customfiletype} repository to this product, and upload the image to the repository.
+For more information, see {ContentManagementDocURL}Importing_Individual_ISO_Images_and_Files_content-management[Importing individual ISO images and files] in _{ContentManagementDocTitle}_.
+. In the {ProjectWebUI}, navigate to *Configure* > *Host Groups*.
+. Select the host group that you want to use.
+. Click the *Parameters* tab, and then click *Add Parameter*.
+. Create a parameter named `kickstart_liveimg` with type *string* and set the value to the URL or a relative path of the image location, for example:
+* URL: `https://_{smartproxy-example-com}_/pulp/content/_My_Organization_Label_/Library/custom/_My_Product_Label_/_My_Repository_Label_/_My_Image_Name_.tar`
+* Relative path: `custom/_My_Product_Label_/_My_Repository_Label_/_My_Image_Name_.tar`
```

Comment:
@ShimShtein Does it mean that this example is wrong?

Reply:
Relative path + kt_activation_keys defined

Relative path example: my_image_name.tar
`_foreman-url_/pulp/content/_LCE_/_content_view_label_/$param`